### PR TITLE
Update tr.coffee

### PR DESCRIPTION
**Pull request by requizm**

### CODE REVIEW

**Comment by requizm** [comment]
Just added some new translations.

**Comment by Bryukh** [comment]
@requizm Could you sign our [CLA](https://contributors.codecombat.com/cla) so we can accept your PR?

**Comment by requizm** [comment]
I did sign.

**Comment by requizm** [comment]
I did sign.

**Comment by Bryukh** [comment]
@jmif could you look at checks here? Is it connected with copy-i18n ?

File: app/locale/tr.coffee [hunk 1]
```diff
@@ -260,58 +260,58 @@ module.exports = nativeDescription: "Türkçe", englishDescription: "Turkish", t
     campaign_multiplayer_description: "Diğer oyuncularla kafa kafaya verip kodlamak için..."
 #    brain_pop_done: "You’ve defeated the Ogres with code! You win!"
 #    brain_pop_challenge: "Challenge yourself to play again using a different programming language!"
-#    replay: "Replay"
+    replay: "Tekrar"
 #    back_to_classroom: "Back to Classroom"
 #    teacher_button: "For Teachers"
 #    get_more_codecombat: "Get More CodeCombat"
 
 #  code:
```
Comment (AndrewJakubowicz) [inline app/locale/tr.coffee]: Error is due to this comment. Update with:
```suggestion
  code:
```

File: app/locale/tr.coffee [hunk 2]
```diff
@@ -991,29 +991,29 @@ module.exports = nativeDescription: "Türkçe", englishDescription: "Turkish", t
 #    restricted_to_certain_heroes: "Only certain heroes can play this level."
 
 #  skill_docs:
```
Comment (AndrewJakubowicz) [inline app/locale/tr.coffee]: Error is also due to this comment. Update with:
```suggestion
  skill_docs:
```

**Pull request merged**